nix/pkgs/static-runc.nix

```diff
@@ -0,0 +1,22 @@
+{ runc, fetchFromGitHub }:
+runc.overrideAttrs  (old: rec {
+  name = "runc-${version}";
+  # latest version seems not to work with agent?
+  version = "1.0.0-rc10";
+
+  src = fetchFromGitHub {
+    owner = "opencontainers";
+    repo = "runc";
+    rev = "v${version}";
+    sha256 = "sha256-8ylYXElsHZbxwvWCLMVNrvv0xe9T0vRJPSkVVOTOI14=";
+  };
+
+  # static linking
+  CGO_ENABLED = 0;
+  preInstall = ''
+    # don't wrap the binary with a shell script
+    wrapProgram() {
+      :
+    }
```

Comment:
what does `:` do?

Reply:
Functions and if statement must contain at least one statement. Since I want an empy function I need therefore add `:`, which is a no-op.